view/concept-shared.twig

```diff
@@ -39,7 +39,7 @@
       {% spaceless %}
       <div class="row{% if concept.type == 'skosext:DeprecatedConcept' %} deprecated{% endif %}">
         <div class="property-label property-label-pref">
-          <span class="versal">{{ 'skos:prefLabel'|trans|upper }}</span>
+          <span class="versal">{% if concept.preferredSubpropertyLabelTranslation(request.lang) %}{{ concept.preferredSubpropertyLabelTranslation(request.lang)|upper }}{% else %}{{ 'skos:prefLabel'|trans|upper }}{% endif %}</span>
```

Comment:
The template calls `preferredSubpropertyLabelTranslation` twice; this might be bad for performance. It could be better to assign a variable so that only a single call is necessary.